### DISCUSSION TITLE
Hye I am new

### DISCUSSION THREAD

**Discussion by ptiwari6239** [WasmEdge/WasmEdge#2430] [General]
hye everyone , i am new to open source and wanted to start contributing towards this project , i am learning Devops and cloud . Also i am good in java and python languages 
can you guys please share how do i start contributing towards this project or any project related to cncf and what technologies should i learn

**Comment by alabulei1** [comment]
Hi @ptiwari6239 

Thanks for reaching out. As you're an expert in Java, maybe you can help with WasmEdge's Java SDK: https://github.com/WasmEdge/WasmEdge/pull/2297

As you're learning DevOps, you can also try running WasmEdge with K8s: https://wasmedge.org/book/en/use_cases/kubernetes.html Let us know if you have any questions.
